bool start
bool stop
int32 x
int32 y
int32 width
int32 height
---
int32 x
int32 y
int32 width
int32 height